app/Frame.js

```diff
@@ -3,6 +3,110 @@ import { AppContext } from "./context/AppContext";
 import { BrowserRouter, Route, Switch } from "react-router-dom";
 import { useMediaPredicate } from "react-media-hook";
 
+/* CLDR Data */
+
+import likelySubtags from 'cldr-core/supplemental/likelySubtags.json';
+import currencyData from 'cldr-core/supplemental/currencyData.json';
+import weekData from 'cldr-core/supplemental/weekData.json';
+
+import bgNumbers from 'cldr-numbers-full/main/bg/numbers.json';
+import bgLocalCurrency from 'cldr-numbers-full/main/bg/currencies.json';
+import bgCaGregorian from 'cldr-dates-full/main/bg/ca-gregorian.json';
+import bgDateFields from'cldr-dates-full/main/bg/dateFields.json';
+
+import deNumbers from 'cldr-numbers-full/main/de/numbers.json';
+import deLocalCurrency from 'cldr-numbers-full/main/de/currencies.json';
+import deCaGregorian from 'cldr-dates-full/main/de/ca-gregorian.json';
+import deDateFields from'cldr-dates-full/main/de/dateFields.json';
+
+import esNumbers from 'cldr-numbers-full/main/es/numbers.json';
+import esLocalCurrency from 'cldr-numbers-full/main/es/currencies.json';
+import esCaGregorian from 'cldr-dates-full/main/es/ca-gregorian.json';
+import esDateFields from'cldr-dates-full/main/es/dateFields.json';
+
+import faNumbers from 'cldr-numbers-full/main/fa/numbers.json';
+import faLocalCurrency from 'cldr-numbers-full/main/fa/currencies.json';
+import faCaGregorian from 'cldr-dates-full/main/fa/ca-gregorian.json';
+import faDateFields from'cldr-dates-full/main/fa/dateFields.json';
+
+import frNumbers from 'cldr-numbers-full/main/fr/numbers.json';
+import frLocalCurrency from 'cldr-numbers-full/main/fr/currencies.json';
+import frCaGregorian from 'cldr-dates-full/main/fr/ca-gregorian.json';
+import frDateFields from'cldr-dates-full/main/fr/dateFields.json';
+
+import gbNumbers from 'cldr-numbers-full/main/en-GB/numbers.json';
+import gbLocalCurrency from 'cldr-numbers-full/main/en-GB/currencies.json';
+import gbCaGregorian from 'cldr-dates-full/main/en-GB/ca-gregorian.json';
+import gbDateFields from'cldr-dates-full/main/en-GB/dateFields.json';
+
+import usNumbers from 'cldr-numbers-full/main/en/numbers.json';
+import usLocalCurrency from 'cldr-numbers-full/main/en/currencies.json';
+import usCaGregorian from 'cldr-dates-full/main/en/ca-gregorian.json';
+import usDateFields from'cldr-dates-full/main/en/dateFields.json';
+
+import zhNumbers from 'cldr-numbers-full/main/zh/numbers.json';
+import zhLocalCurrency from 'cldr-numbers-full/main/zh/currencies.json';
+import zhCaGregorian from 'cldr-dates-full/main/zh/ca-gregorian.json';
+import zhDateFields from'cldr-dates-full/main/zh/dateFields.json';
+
+import { IntlProvider, load, loadMessages, LocalizationProvider } from '@progress/kendo-react-intl';
```

Comment:
This import should not be in the file you create to have all of the imports (I dont think). Can still be in the Frame,js on it's own? Or does it make sense to extract it as well? I'll let you make that choice.